src/constants/common.js

```diff
@@ -39,8 +39,19 @@ module.exports = {
 	RESOURCE_CREATOR: 'resource_creator',
 	ADMIN_ROLE: 'admin',
 	ORG_ADMIN_ROLE: 'org_admin',
-	NO_OF_MINUTES: 30,
-	NO_OF_EXPIRY_TIME: 60,
+	CERTIFICATE_PATH: 'certificate/',
+	LOGO_PATH: 'certificate_logo/',
+	SIGNATURE_PATH: 'certfile_signature/',
+	BASETEMPLATE_PATH: 'certfile_basetemplate/',
```

Comment:
@adithyadinesh0412 BASETEMPLATE_PATH and some of them are not using anywhere can you please remove which ever is not used

Reply:
sorry not Adithya @ankit1shahu

Reply:
Using this @priyanka-TL